### PR TITLE
Including documentation workflow

### PR DESCRIPTION
**Pull request by asettle**

### CODE REVIEW

**Comment by hughsaunders** [comment]
skip

**Comment by hughsaunders** [comment]
skip ci

**Comment by hughsaunders** [comment]
skip ci

**Comment by asettle** [comment]
recheck

**Comment by git-harry** [comment]
@asettle based on the conversation in today's meeting, please would you update https://github.com/rcbops/rpc-openstack/blob/master/.github/ISSUE_TEMPLATE.md to include a mandatory docs check box as suggested by @mancdaz 

In addition, would you mind adding some additional notes regarding the alternative approach of raising an issue.

**Comment by git-harry** [comment]
Also, remember to squash your commits.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -43,6 +43,30 @@ When creating an issue, please ensure you include the following information
...
+
+* There are changes to the way playbooks are run/named
+* Changes to the documented configuration files
+* New feature
+* Change that requires you (as the developer) to take action
+* Update impact changes
+
+## how to document
+
+Docs work in:[link](https://github.com/rackerlabs/docs-rpc "docs-rpc")
```
Comment (mancdaz) [inline CONTRIBUTING.md]: This link is weird.

Maybe:

The doc repo is [here](https://github.com/rackerlabs/docs-rpc)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -43,6 +43,30 @@ When creating an issue, please ensure you include the following information
...
+
+1. See [link](https://github.com/rackerlabs/docs-rpc/blob/master/GITHUBING.rst "Docs workflow") to setup for contribution.
```
Comment (mancdaz) [inline CONTRIBUTING.md]: same.

See [here](https://github.com/rackerlabs/docs-rpc/blob/master/GITHUBING.rst "Docs workflow") for information on getting yourself set up to contribute

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -43,6 +43,30 @@ When creating an issue, please ensure you include the following information
...
+2. Make changes in own branch off fork.
+3. Write relevant commit message.
+4. Push up to docs-rpc.
+5. Tag doc team for review. See [link](https://github.com/rackerlabs/docs-rackspace/blob/master/doc/contributor-collateral/docteam-FAQ.rst "Documentation team FAQ") for clarification on who to contact.
```
Comment (mancdaz) [inline CONTRIBUTING.md]: same

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -43,6 +43,30 @@ When creating an issue, please ensure you include the following information
...
+6. Await merge. Respond to any questions/comments/concerns.
+
+Any further questions about our github processes, contributor guidelines, or FAQ about the docs and team, refer to: [link](https://github.com/rackerlabs/docs-rackspace/tree/master/doc/contributor-collateral "Contributor collatoral")
```
Comment (mancdaz) [inline CONTRIBUTING.md]: same

**Comment by asettle** [comment]
@mancdaz - see above :)

**Comment by mancdaz** [comment]
@asettle pls squash commits

**Comment by asettle** [comment]
I AM SQUASHING

**Comment by hughsaunders** [comment]
👍

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -43,6 +43,38 @@ When creating an issue, please ensure you include the following information
...
+* There are changes to the way playbooks are run/named
+* Changes to the documented configuration files
+* New feature
+* Change that requires you (as the developer) to take action
+* Update impact changes
+
+## how to document
+
+Docs work in:[docs-rpc](https://github.com/rackerlabs/docs-rpc "docs-rpc")
+
```
Comment (mancdaz) [inline CONTRIBUTING.md]: Does this mean 'the documentation work is all done in [this](https://github.com/rackerlabs/docs-rpc) repo' ?

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -43,6 +43,38 @@ When creating an issue, please ensure you include the following information
...
+## Documentation impact
+
+## documenting your change
+
+When noting there is a doc impact on your change, you have two choices:
+
+1. Submit an issue to [Docs issues](https://github.com/rackerlabs/docs-rpc/issues "Docs issues")
+OR
+2. Contribute your own doc change to [docs-rpc](https://github.com/rackerlabs/docs-rpc "docs-rpc") and tag a relevant docs person for review and merge.
+
```
Comment (mancdaz) [inline CONTRIBUTING.md]: you can remove the quoted bit inside the brackets

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -43,6 +43,38 @@ When creating an issue, please ensure you include the following information
 6. Each time a PR is merged, the associated branch is deleted from ```rpc-openstack```.
 7. When all PRs are completed the issue is then closed.
 
+## Documentation impact
+
+## documenting your change
+
+When noting there is a doc impact on your change, you have two choices:
+
+1. Submit an issue to [Docs issues](https://github.com/rackerlabs/docs-rpc/issues "Docs issues")
```
Comment (mancdaz) [inline CONTRIBUTING.md]: you can remove the quoted bit inside the brackets

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -43,6 +43,38 @@ When creating an issue, please ensure you include the following information
...
+## how to document
+
+Docs work in:[docs-rpc](https://github.com/rackerlabs/docs-rpc "docs-rpc")
+
+1. See [Docs workflow](https://github.com/rackerlabs/docs-rpc/blob/master/GITHUBING.rst "Docs workflow") to setup for contribution.
+2. Make changes in own branch off fork.
+3. Write relevant commit message.
+4. Push up to docs-rpc.
+5. Tag doc team for review. See [Documentation team FAQ](https://github.com/rackerlabs/docs-rackspace/blob/master/doc/contributor-collateral/docteam-FAQ.rst "Documentation team FAQ") for clarification on who to contact.
+6. Await merge. Respond to any questions/comments/concerns.
```
Comment (mancdaz) [inline CONTRIBUTING.md]: you can remove the quoted bit inside the brackets

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -43,6 +43,38 @@ When creating an issue, please ensure you include the following information
...
+
+Any further questions about our github processes, contributor guidelines, or FAQ about the docs and team, refer to: [Contributor collatoral](https://github.com/rackerlabs/docs-rackspace/tree/master/doc/contributor-collateral "Contributor collatoral")
```
Comment (mancdaz) [inline CONTRIBUTING.md]: you can remove the quoted bit inside the brackets

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -43,6 +43,38 @@ When creating an issue, please ensure you include the following information
 6. Each time a PR is merged, the associated branch is deleted from ```rpc-openstack```.
 7. When all PRs are completed the issue is then closed.
 
+## Documentation impact
+
+## documenting your change
+
```
Comment (mancdaz) [inline CONTRIBUTING.md]: this should be ### to indicate the next level of heading

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -43,6 +43,38 @@ When creating an issue, please ensure you include the following information
...
+When noting there is a doc impact on your change, you have two choices:
+
+1. Submit an issue to [Docs issues](https://github.com/rackerlabs/docs-rpc/issues)
+OR
+2. Contribute your own doc change to [docs-rpc](https://github.com/rackerlabs/docs-rpc) and tag a relevant docs person for review and merge.
+
+## what to document
```
Comment (mancdaz) [inline CONTRIBUTING.md]: this should be ### to indicate the next level of heading

File: CONTRIBUTING.md [hunk 12]
```diff
@@ -43,6 +43,38 @@ When creating an issue, please ensure you include the following information
...
+
+Document changes if:
+
+* There are changes to the way playbooks are run/named
+* Changes to the documented configuration files
+* New feature
+* Change that requires you (as the developer) to take action
+* Update impact changes
+
+## how to document
```
Comment (mancdaz) [inline CONTRIBUTING.md]: this should be ### to indicate the next level of heading

**Comment by mancdaz** [comment]
👍 please remember to always squash commits

File: CONTRIBUTING.md [hunk 13]
```diff
@@ -43,6 +43,39 @@ When creating an issue, please ensure you include the following information
...
+
+### how to document
+
+The documentation work is all done in [this](https://github.com/rackerlabs/docs-rpc) repo.
+
+1. See the [Docs workflow](https://github.com/rackerlabs/docs-rpc/blob/master/GITHUBING.rst) to setup for contribution and
+   [here](https://github.com/rackerlabs/docs-rpc/blob/master/CONTRIBUTING.rst) for our Contributor guidelines.
+2. Make changes in own branch off fork.
+3. Write relevant commit message.
+4. Push up to docs-rpc.
```
Comment (git-harry) [inline CONTRIBUTING.md]: Is step 4 the create pull request step?

Steps 2, 3 and 4 seem like an inconsistent level of detail and they are already covered in the links in step 1.

Maybe remove 2-4 and amend step 5 by prepending it with 'Once you have created a pull request, '.

**Pull request merged**